tests/Persistence/Mapping/ClassMetadataFactoryTest.php

```diff
@@ -49,15 +50,15 @@ public function testSetGetCache(): void
 
     public function testGetMetadataFor(): void
     {
-        $metadata = $this->cmf->getMetadataFor('stdClass');
+        $metadata = $this->cmf->getMetadataFor(stdClass::class);
 
-        self::assertTrue($this->cmf->hasMetadataFor('stdClass'));
+        self::assertTrue($this->cmf->hasMetadataFor(stdClass::class));
     }
 
     public function testGetMetadataForAbsentClass(): void
     {
         $this->expectException(MappingException::class);
-        $this->cmf->getMetadataFor(__NAMESPACE__ . '\AbsentClass');
+        $this->cmf->getMetadataFor(Foo::class);
```

Comment:
Staying with the former wouldn't produce a new PHPStan's baseline entry

Reply:
In isolation, yes, but there are other occurrences of `Foo::class` in that file.